cogify/README.md

```diff
@@ -40,3 +40,9 @@ python3 ERA5/fetch.py
 # Generate the cog
 python3 run.py -f download.nc -c ERA5
 ```
+
+
+AWS Provisioning:
+To function as a lambda task, the Lambda function's execution role needs to be given permission to AWS PutObject.
+
+- Add permission policy to the Lambda's execution role to allow `s3:PutObject`. This is because this Cogify lambda needs to access an S3 Bucket to write COG files
```

Comment:
I would guess that this permission policy is something we can add programmatically through CDK, but for now you can add an issue to this repo to do that.

Reply:
i was wondering about this, ill do some research and add an issue outlining what needs to be done